cmd/influx/pkg.go

```diff
@@ -1112,6 +1123,7 @@ func (d *diffPrinter) Render() {
 
 	d.setFooter()
 	d.writer.Render()
+	fmt.Fprintln(d.w)
```

Comment:
? does it do anything

Reply:
I forgot to add this in last PR. What it does is extend the margin around the table print so that the table has a buffer of 1 newline. Makes it so that when printing a table it takes care of styling itself, including margins 🤷‍♂. Similar to how css has margin but way more hacky lol

before newline buffer:
![image](https://user-images.githubusercontent.com/17263167/78185800-7ccff380-7420-11ea-807d-b2a73a1c264a.png)

with newline buffer:
![image](https://user-images.githubusercontent.com/17263167/78185769-73468b80-7420-11ea-924c-1c32e1fb0ecd.png)